app/views/catalog/numismatics.html.erb

```diff
@@ -0,0 +1,48 @@
+<h3 class="title">Numismatics</h3>
+<div class="row">
+  <div class="col-md-12">
+    <div class="panel-body">
+      <% @numismatic_decorator.metric_labels_first.each_with_index do |name, index| %>
```

Comment:
Rather than add this label to the NumismaticIssue decorator, it would probably be better to just enumerate them here:

```
<% ["Accessions", "Firms", "Monograms"].each_with_index do |name, index| %>
```

Or, even better, create a partial to reduce some of the code repetition. Like we do in in the `home_text` view:
https://github.com/pulibrary/figgy/blob/master/app/views/catalog/_home_text.html.erb#L10-L12

Reply:
I don't want to have the array in the view this is why I added in the decorator since we use decorators for the views.

Reply:
I understand that is a method specific to the NumismaticIssue decorator and not to the NumismaticDecorator since we have not refactored them yet and we dont have a NumismaticDecorator but isn't it better to be outside of the view?

Reply:
I will add the partials.

Reply:
I will also keep the strings in the view as it was suggested from @eliotjordan and @tpendragon